db/migrate/20180216103915_add_fields_to_volunteer_ops.rb

```diff
@@ -0,0 +1,9 @@
+class AddFieldsToVolunteerOps < ActiveRecord::Migration[5.1]
+  def change
+    add_column :volunteer_ops, :role_description, :string
+    add_column :volunteer_ops, :skills_needed, :string
+    add_column :volunteer_ops, :when_volunteer_needed, :string
```

Comment:
@tansaku  kindly confirm if this is a `boolean` or a `string`. What kind of information is required

Reply:
I think when volunteer needed would be a date

Reply:
@sigu and @tansaku Thanks, let me change that.